honeycombio/resource_derived_column.go

```diff
@@ -55,6 +55,13 @@ func resourceDerivedColumnImport(ctx context.Context, d *schema.ResourceData, i
 	dataset := strings.Join(idSegments[0:len(idSegments)-1], "/")
 	alias := idSegments[len(idSegments)-1]
 
+	client := meta.(*honeycombio.Client)
+	derivedColumn, err := client.DerivedColumns.GetByAlias(ctx, dataset, alias)
+	if err != nil {
+		return nil, err
+	}
+
+	d.SetId(derivedColumn.ID)
```

Comment:
Same here, we don't need to get and store the ID because it is set by `resourceDerivedColumnRead`.